opensea-subgraph-dev/web/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
Let's get rid of all these web stuff for now. If we have a fully fledged website later, we can push them again